### ISSUE TITLE
When running individual unit test using WordSpec, getting false positive instead of failing test

### ISSUE THREAD

**Issue by ataronet** [kotest/kotest#2319]
**Which version of Kotest are you using**
Plugin version 1.1.36-IC-2021.1
kotest version 4.5.0
kotlin version 1.5.0

Try the following test, see that test 2 should fail. If you run the test with the green arrow near the `TestMe`, it does fail. But if you use the green arrow next to `"test 2 should fail"` the test report as pass and it is green.
```
class TestMe : WordSpec() {
    init {
        "tests numbers" should {
            "test 1" {
                val i = 1
                i shouldBe 1
            }
            "test 2 should fail" {
                val i = "3"
                i shouldBe "2"
            }
        }
    }
}
```
![image](https://user-images.githubusercontent.com/27952413/121501261-68bf5300-ca22-11eb-8245-603d5ae9d6bd.png)

**Comment by sksamuel** [comment]
I get the same issue. It's because the test path isn't being picked up properly and it can't find the subtest.
You notice that when you run a subtest it's actually not running the tests at all.

Workaround - use another type of spec.
Will include a fix in 4.6.1

**Comment by ataronet** [comment]
Yes, I noticed it doesn't run it at all.
Thank you @sksamuel. I will use the workaround till it is fixed.

**Comment by sksamuel** [comment]
Fixed in 4.6.1
